src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -326,21 +328,20 @@ The `description` property describes the purpose of your app.
 
 The `screenshots` property is an array of image objects representing your app
 in common usage scenarios. Each object must include the `src`, a `sizes`
-property, and the `type` of image.
+property, and the `type` of image.  The `form_factor` property is optional.
+It can be set either to `"wide"` for screenshots applicable to wide screens
```

Comment:
Can we add images with examples? I think that would help to explain 'narrow' vs 'wide' and will show the pretty interface you get with RIUI

Reply:
I'll do that. Good suggestion!

Reply:
FYI I've started https://github.com/GoogleChromeLabs/squoosh/pull/1283 so that I can use squoosh.app screenshots

Reply:
And it's fixed. See latest commit.

Reply:
@tropicadri Does that look good now?

Reply:
@tropicadri (gentle ping)